packages/webapp/src/routes/developer/check-and-submit.js

```diff
@@ -0,0 +1,160 @@
+  const allHabitats = formatHabitatDetails(habitatDetails)
+  const habitats = (allHabitats || []).map(item => {
+    return [
+      {
+        classes: 'govuk-body-s govuk-!-display-block govuk-!-margin-top-0 govuk-!-margin-bottom-0',
+        value: item?.header
+      },
+      {
+        classes: 'govuk-body-m govuk-!-display-block govuk-!-margin-top-0 govuk-!-margin-bottom-0',
+        value: item?.description
+      },
+      {
+        classes: 'govuk-body-s govuk-!-display-block govuk-!-margin-top-0 govuk-!-margin-bottom-0 app-secondary-text-colour',
+        value: item?.condition
+      },
+      {
+        classes: false,
+        value: item?.amount
+      }
+    ]
+  })
+
+  const developerIsAgent = session.get(constants.redisKeys.DEVELOPER_IS_AGENT) === constants.APPLICANT_IS_AGENT.YES
+  const developerIsLandowner = session.get(constants.redisKeys.DEVELOPER_LANDOWNER_OR_LEASEHOLDER) === constants.DEVELOPER_IS_LANDOWNER_OR_LEASEHOLDER.YES
+  const fileLocation = session.get(constants.redisKeys.DEVELOPER_PLANNING_DECISION_NOTICE_LOCATION)
+  const planningDecisionNoticeFileName = fileLocation === null ? '' : path.parse(fileLocation).base
+  const clientType = developerIsAgent
+    ? session.get(constants.redisKeys.DEVELOPER_CLIENT_INDIVIDUAL_ORGANISATION)
+    : session.get(constants.redisKeys.DEVELOPER_LANDOWNER_TYPE)
+  const clientsName = session.get(constants.redisKeys.DEVELOPER_CLIENTS_NAME)?.value
+  const clientTypeNameisAgent = clientType ? initialCapitalization(clientType) : null
+  const clientTypeNameisLandowner = getClientTypeNameisLandowner(clientType)
+  const { subject } = getApplicantContext(request.auth.credentials.account, session)
+
+  return {
+    applicantInfo: {
+      actingForClient: developerIsAgent ? 'Yes' : 'No',
+      actingForClientChangeUrl: constants.routes.DEVELOPER_AGENT_ACTING_FOR_CLIENT,
+      confirmed: session.get(constants.redisKeys.DEVELOPER_DEFRA_ACCOUNT_DETAILS_CONFIRMED),
+      confirmedChangeUrl: constants.routes.DEVELOPER_CHECK_DEFRA_ACCOUNT_DETAILS,
+      accountDetails: session.get(constants.redisKeys.DEVELOPER_DEFRA_ACCOUNT_DETAILS_CONFIRMED) ? `Yes, apply as ${subject}` : 'No',
+      landownerOrLeaseHolderTitle: developerIsAgent ? 'Client is a landowner or leaseholder' : 'Applying as landowner or leaseholder',
+      landownerOrLeaseholder: developerIsLandowner ? 'Yes' : 'No',
+      landownerOrLeaseholderChangeUrl: constants.routes.DEVELOPER_LANDOWNER_OR_LEASEHOLDER,
+      clientTypeTitle: developerIsAgent ? 'Client is an individual or organisation' : 'Applying as individual or organisation',
+      clientType: developerIsAgent ? clientTypeNameisAgent : clientTypeNameisLandowner,
+      clientTypeChangeUrl: developerIsAgent ? constants.routes.DEVELOPER_CLIENT_INDIVIDUAL_ORGANISATION : constants.routes.DEVELOPER_APPLICATION_BY_INDIVIDUAL_OR_ORGANISATION,
+      clientsName: clientsName ? `${clientsName?.firstName} ${clientsName?.lastName}` : '',
```

Comment:
This needs another conditional as the client could be individual or organisation

Reply:
done